### PR TITLE
vmctl: update VictoriaMetrics migration section

### PR DESCRIPTION
**Pull request by hagen1778**
Remove unnecessary information to simplify the description and tips.

### CODE REVIEW

File: app/vmctl/README.md [hunk 1]
```diff
@@ -765,109 +761,59 @@ Requests to make: 9 / 9 [██████████████████
...
-2023/04/11 10:19:45 VictoriaMetrics importer stats:
-  time spent while importing: 2m30.813841541s;
-  total bytes: 1.8 GB;
-  bytes/s: 11.7 MB;
-  requests: 1;
-  requests retries: 0;
-2023/04/11 10:19:45 Total time: 2m30.814721125s
-```
+_To disable explore phase and switch to the old way of data migration via single connection use 
+`--vm-native-disable-retries` cmd-line flag. Please note, in this mode vmctl won't be able to retry failed requests._
```
Comment (dmitryk-dk) [inline app/vmctl/README.md:766]: there is a typo `_`

File: docs/vmctl.md [hunk 2]
```diff
@@ -769,109 +765,59 @@ Requests to make: 9 / 9 [██████████████████
...
-2023/04/11 10:19:45 Import finished!
-2023/04/11 10:19:45 VictoriaMetrics importer stats:
-  time spent while importing: 2m30.813841541s;
-  total bytes: 1.8 GB;
-  bytes/s: 11.7 MB;
-  requests: 1;
-  requests retries: 0;
-2023/04/11 10:19:45 Total time: 2m30.814721125s
-```
+_To disable explore phase and switch to the old way of data migration via single connection use
```
Comment (dmitryk-dk) [inline docs/vmctl.md:769]: same typo `_`

**Review by dmitryk-dk — COMMENTED** [review]
Looks good! Please check the comments

File: app/vmctl/README.md (hunk 1 above)
Comment (hagen1778) [reply app/vmctl/README.md:766]: It is not a typo, it is `_cursive_` style in markdown

File: docs/vmctl.md (hunk 2 above)
Comment (hagen1778) [reply docs/vmctl.md:769]: See https://github.com/VictoriaMetrics/VictoriaMetrics/pull/4310#discussion_r1193894554

File: app/vmctl/README.md (hunk 1 above)
Comment (dmitryk-dk) [reply app/vmctl/README.md:766]: ok, I missed first `_` :)

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Pull request merged**